src/app/onion/dashboard/dashboard.component.ts

```diff
@@ -133,6 +135,13 @@ export class DashboardComponent implements OnInit, OnDestroy {
       const filters = {status: Array.from(this.filters.keys())};
       this.learningObjects = await this.getLearningObjects(filters);
     });
+
+    // if the user is trying to edit a released learning object, then get the status and alert the user
+    this.route.queryParams.subscribe(queryParams => {
```

Comment:
This subscription should follow the takeUntil pattern we've used throughout the system to prevent leaving subscriptions open after components have been destroyed.

Reply:
Subscription now uses the takeUntil pattern.